catalog/view/stylesheet/securesubmit.css

```diff
@@ -164,4 +175,12 @@
   width:310px;
   background-size:310px 40px;
 }
+.form-group:nth-child(2),
+.form-group:nth-child(3) {
+  width:100%;
+  display:block;
+}
+.form-group:nth-child(3) {
+  float:none;
+}
```

Comment:
These statements should be be altered so that they are specific to Heartland's elements. As it stands, they could affect other elements on the page.

Reply:
css updated